controllers/agentaction_controller.go

```diff
@@ -302,6 +312,16 @@ func (r *AgentActionReconciler) createConfigSecret(ctx context.Context, log logr
 			GenerateName: action.Name + "-",
 			Namespace:    action.Namespace,
 			Labels:       labels,
+			OwnerReferences: []metav1.OwnerReference{
```

Comment:
There's a bunch of notes throughout the operator stating we don't use this because it has panics - I can switch it and see if the tests get upset though - maybe it's an old note?

Reply:
I think it is an old note.  When changing this, locally it didn't do anything bad.

Reply:
kk will update